exercises/Python_Principles_4.ipynb

```diff
@@ -187,22 +178,19 @@
     "\n",
     "To use an if statement it's necessary to understand some of the operators we have seen in a different section. So make sure you review those.\n",
     "\n",
-    "The syntax for an `if` statement is the following:\n",
-    "```\n",
-    "if <condition>:\n",
-    "\tcode\n",
-    "```\n",
+    "The syntax for an `if` statement \n",
     "\n",
-    "Here we are saying that the condition that needs to be met in order for code below the if statement is run. The condition is usually an expression or a datatype that is truthy or falsy. We can use any operators like `==`, `<`, `and` we like when we use if statements.\n",
+    "`if` expression:\n",
+    "\tcode\n",
     "\n",
-    "It's important to indent the code below the `if` as this tells Python the code block is within the if statement (Remember back to the first exercise on this).\n",
+    "It's important to ident the code below the `if` as this tells Python the code block is within the if statement.\n",
```

Comment:
@AlexTeslenko Did you mean to add ident instead of removing indent?

Reply:
fixed this one